.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.nvmrc
```

Comment:
Nit: it's customary to add this type of thing to your workstation's global `.gitignore` because `nvm` is not a prerequisite of this lib

Reply:
Thanks for this-- I had no idea that .gitignores above a repository root would be respected, but on reflection I can't think of why they wouldn't be. I'll remove this.